_data/education.yml

```diff
@@ -220,6 +220,13 @@ websites:
       twitter: ISACANews
       facebook: ISACAHQ
 
+    - name: KnowBe4 
+      url: https://training.knowbe4.com/
```

Comment:
How come you used training subdomain and not the main one or www?

Reply:
Once you create an account with KnowBe4 you are redirected to https://training.knowbe4.com and is where the MFA is configured. I thought that was most appropriate as the url considering.

Reply:
@arnydo no, we list the main url here. Please change the url to `https://www.knowbe4.com/`